task/k8s/resources/resource_tx_pod.go

```diff
@@ -0,0 +1,146 @@
+package resources
```

Comment:
Consider renaming `resource_tx_pod.go` to `resource_transfer_pod.go` so it can't be mistaken with _transaction_ nor _transmission_. Blockchain and embedded system developers will be extremely thankful. 😄

Reply:
Resolved with https://github.com/iterative/terraform-provider-iterative/pull/669/commits/97cb39e774d9d00790402afeaa544fd19f565a37